pkgs/development/python-modules/aiopg/default.nix

```diff
@@ -0,0 +1,41 @@
+{ lib
+, async-timeout
+, buildPythonPackage
+, fetchFromGitHub
+, psycopg2
+, pythonOlder
+}:
```

Comment:
Slight nitpick, but convention is that the ordering for this typically goes:
* lib
* buildPythonPackage
* pythonOlder
* fetchers
* imports (e.g. async-timeout, psycopg2)